package.json

```diff
@@ -22,7 +22,8 @@
     "test-node": "gulp test",
     "test-browser": "mockttp -c karma start",
     "build": "gulp build",
-    "prepublish": "npm test && npm run build",
+    "prepublish": "require-npm4-to-publish",
+    "prepublishOnly": "npm run build",
```

Comment:
How about moving away from prepublish & prepublishOnly and going for `prepack`?
As mentioned in [this thread](https://www.flowdock.com/app/rulemotion/r-beginners/messages/66691) the deploy job uses resinci/npm-x86_64-alpine-node10:master which has npm v6.1.0 :)
I guess in that case we should also have a require-npm5-to-publish module or drop the prepublish :thinking:

Reply:
As of npm@6.4.1 prepublish still works w/ a deprecated warning.